scripts/gpg-share.sh

```diff
@@ -61,20 +63,11 @@ check_pwd () {
 }
 
 
-check_dependencies
-check_pwd
-
-# get the version
-VERSION=$(npm pkg get version | sed 's/"//g')
-
-# build the archive name
-ARCHIVE="keys-${VERSION}.tar"
-
-
 # TODO: documentation
 check_file () {
   [ -z "$1" ] && {
     log_error "Please give a file to share as the first argument"
```

Comment:
*same as the other error msg*
Now that flags has been introduced you should change this to :
```diff
-    log_error "Please give a file to share as the first argument"
+    log_error 'Please give a file to share with `-f` flag'

Reply:
should be fixed in ca775a9